codalab/apps/web/static/js/vendor/jquery-editable-poshytip.js

```diff
@@ -2187,7 +2186,7 @@ Makes editable any HTML element on the page. Applied as jQuery method.
         @type string
         @default 'Empty'
         **/         
-        emptytext: 'Empty',
+        emptytext: '{empty}',
```

Comment:
Italics would be a nice way to differentiate from a title that is "(none)"
On Sun, Nov 1, 2015 at 4:33 PM Percy Liang <notifications@github.com> wrote:

> In codalab/apps/web/static/js/vendor/jquery-editable-poshytip.js
> <https://github.com/codalab/codalab/pull/1374#discussion_r43590976>:
>
> > @@ -2187,7 +2186,7 @@ Makes editable any HTML element on the page. Applied as jQuery method.
> >          @type string
> >          @default 'Empty'
> >          **/
> > -        emptytext: 'Empty',
> > +        emptytext: '{empty}',
>
> Can't you use ampersand-lt-semicolon?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codalab/codalab/pull/1374/files#r43590976>.
>